pkgs/by-name/xm/xmoji/package.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  fontconfig,
+  harfbuzz,
+  libpng,
+  xorg,
```

Comment:
```suggestion
  xcbutil,
  libXcursor,
  xcbutilimage,
```

Reply:
Is there any recommendation for abolishing `xorg` this way?

Reply:
Makes it easier to override, as you don't have to override the whole package set.
And CI complained about formatting, which was obviated by not passing xorg at all.